build.gradle

```diff
@@ -37,6 +38,7 @@ buildscript {
         classpath ("com.vanniktech:gradle-android-junit-jacoco-plugin:$gradleJunitJacocoPluginVersion")
         classpath ("gradle.plugin.com.github.spotbugs.snom:spotbugs-gradle-plugin:$gradleSpotbugsPluginVersion")
         classpath ("com.google.gms:google-services:$googleGmsServicesPluginVersion")
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
```

Comment:
nit;
```suggestion
        classpath ("org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version")
```

Reply:
Is there any reason we want to add this here if we only want to use it for the Sample Chat App?